frontend/src/views/ProducerFormPage/AttachmentStep.vue

```diff
@@ -1,6 +1,94 @@
 <template>
-  <h2 class="fr-h6">
+  <h3 class="fr-h6">
+    <v-icon class="mr-1" name="ri-price-tag-2-fill"></v-icon>
+    Étiquetage
+  </h3>
+
+  <DsfrInputGroup>
+    <DsfrFileUpload
+      label="Merci d'ajouter au moins un fichier image ou PDF correspondant à l'étiquetage."
+      :accept="['image/jpeg, image/gif, image/png, application/pdf']"
+      @change="addLabelFiles"
+      v-model="selectedLabelFile"
+    />
+  </DsfrInputGroup>
+
+  <div class="grid grid-cols-12 gap-3">
```

Comment:
Je pense que c'est dès qu'il y a une répétition de styles `grid grid-cols-12 gap-3` dans les classes, ça montre qu'il faut encapsuler dans un nouveau composant. Cf. le refactor sur https://github.com/betagouv/complements-alimentaires/tree/staging/frontend/src/views/ElementPage

Reply:
Même si in fine ça fait plus de components, s'ils sont bien nommés, ça simplifie la lecture du code je pense, comme quand on utilise des fonctions au sein de fonctions

Reply:
Je l'ai fait dans la branche `attachment-step` : https://github.com/betagouv/complements-alimentaires/pull/275/commits/a6e42927068bff629d8300efac469edeb1de9ffe